python/tvm/relay/op/_tensor.py

```diff
@@ -201,11 +201,11 @@ def elemwise_shape_func(attrs, inputs, _):
     return [topi.math.identity(inputs[0])]
 
 register_shape_func("cast", False, elemwise_shape_func)
-register_shape_func("zeros", True, no_data_full_shape_func)
+register_shape_func("zeros", False, full_shape_func)
 register_shape_func("zeros_like", False, elemwise_shape_func)
-register_shape_func("ones", True, no_data_full_shape_func)
+register_shape_func("ones", False, full_shape_func)
```

Comment:
:/ Is this a rebase error?

Reply:
No, I was going through my code and I forgot to change this for the zeros and ones PR -- for the static shape ones and zeros, it is not data dependent since the shape of the input is in attributes, and uses the normal full_shape_func. It passes the tests for static ones and zeros..

Reply:
Perhaps I should just remove the shape func registration for the static ops, it seems like this change didn't affect any behavior.